### DISCUSSION TITLE
People Search Results: User Properties not showing when Show Persona Card (Graph) selected

### DISCUSSION THREAD

**Discussion by cli1810** [microsoft-search/pnp-modern-search#3714] [Q&A]
Hello everyone,

I am using the Search Results webpart for an org people search. I have in my people search some extension attributes and also some special Entra Properties, I am synchonizing per PNP. Here everything is working well.
But, I have some default properties like: Office Location, Department. And when I am activating the "Show Persona Card on hover (LPC)" this properties are dissapearing from my template. Because they are default properties they are showing in the hover card, but not in my template.
When I am activating the default SharePoint hover card, the properties are there but the hover card is not what I want.
I have also added some new RefinableStrings, mapped them to the crawled properties and have used them, but nothing changes.
I am happy about any tip
Thanks in advance
![PeopleSearchIssue](https://github.com/microsoft-search/pnp-modern-search/assets/155969155/65f7ce59-0ac9-4af4-bca9-1d8ebc09b9aa)

**Comment by kasperbolarsen** [comment]
Sorry, you have to pretend I am 5 years old, and explain your issue again. Are you asking how to change your custom PnP Modern Search Layout to display those properties or are you asking how to change the LPC to show or hide some properties?

**Comment by cli1810** [comment]
hehe, neither of your assuptions. 
When I am using the LPC, the properties from my template are just dissapearing in my people layout card.
As soon as I activate the hover without the LPC, the properties are there in my defined card.
I am not changing anything on the template, but with the LPC active, the properties are not showing whatever I do.

**Comment by wobba** [comment]
The LPC card shows fields according to what is documented at https://learn.microsoft.com/en-us/graph/add-properties-profilecard. You cannot create a custom template for it. This is the same card shown in all M365 experiences.

**Reply by cli1810** [reply]
thank you for the response. 
I am talking about the fields in my card layout, NOT the hover card. 
By activating the LPC hover card, the fields in my own defined layout (see screenshot) are just dissapearing. So the LPC is fine, but by activating it, it just crushes my layout. The fields Office, Position are not showing any more.

**Reply by wobba** [reply]
If you can share exact repro on the setup/template, we may be able to take a look.
